index.html

```diff
@@ -0,0 +1,303 @@
+              <div class="description-tonic primary-text">
+                <p>
+                  Hello I’m a software developer! I can help you build a 
+                  product , feature or website Look through some of my 
+                  work and experience! If you like what you see and have 
+                  a project you need coded, don’t hesitate to contact me.
+                </p>
+              </div>
+              <div class="socialnet">
+                <div class="tlsconnect secondary-links">LET'S connect</div>
+                <div class="social-links">
+                  <div class="enlace">
+                    <a href="#" target="_blank">
+                      <img src="./images/twitter.png" alt="twitter" />
+                  </a>
+                  </div>
+                  <div class="enlace">
+                    <a href="#" target="_blank">
+                      <img src="images/Linkedin.png" alt="linkedIn" />
+                  </a>
+                  </div>
+                  <div class="enlace">
+                    <a href="#" target="_blank">
+                      <img src="images/medium.png" alt="medium" />
+                  </a>
+                  </div>
+                  <div class="enlace">
+                    <a href="#" target="_blank">
+                      <img src="images/github.png" alt="git" />
+                  </a>
+                  </div>
+                  <div class="enlace">
+                    <a href="#" target="_blank">
+                      <img src="images/angelite.png" alt="angel" />
+                  </a>
+                  </div>
+                </div>
+              </div>
+              <div class="buttonab">
+                <button type="submit" class="go-buttonab">Get my resume</button>
+              </div>
+            </div>
+          </div>
+          <div class="card-right1">
+            <div class="languages">
+              <div class="wlanguages">Languages</div>
+              
+            </div>
+            <ul class="lenguagelist">
+              <li class="abouljs">
+                <img class="ellipse"/>
+                <h2 class="lanjav">JavaScript</h2>
+              </li>
+              <li class="aboulht">
+                <img class="elihtml"/>
+                <h2 class="lanhtml">HTML</h2>
+              </li>
+              <li class="aboulcss">
+                <img class="elicss"/>
+                <h2 class="lancss">CSS</h2>
+              </li>
+            </ul>
+            <ul class="frm">
+              <li class="wframe">
+                  <h2 class="wgramew">Frameworks</h2>
+                  <img class="sepa"/>   
+              </li>
+              <li class="wskills">
+                  <h2 class="wskillsw">Skills</h2>
+                  <img class="sepa"/>   
+              </li>
+          </ul>
+          </div>
+        </div>
+      </div>
+      <section id="consection">
```

Comment:
- [ ] I kindly suggest that you follow the Figma design closely in designing your contact section:

- Kindly ensure that your inputs are the same width like they are on the figma design
- Kindly ensure that the padding of the inputs(space within) is the same as the Figma design.


|   |  |
| ------------- | ------------- |
| Your Design  | <img width="695" alt="image" src="https://github.com/jovasal/complete-portfolio/assets/52934128/fcba8c81-65b0-4738-89a9-887d933d6c0a">  |
| Figma design | <img width="540" alt="image" src="https://github.com/jovasal/complete-portfolio/assets/52934128/689e2c6f-fda0-4108-9ddf-664edc4d39a0">  |